seeds/z1000_settlementModel.js-

```diff
@@ -0,0 +1,56 @@
+/*****
```

Comment:
why does the filename have a dash at the end of it?

Reply:
It is currently not executed, but I just wanted to preserve it for a while until it's absolutely certain, there won't be any generic settlement models to be seeded. Please, let me know if it would be better to delete the script.

Reply:
I would make a note in the file if you wish to keep it

Reply:
I would suggest making a Note in the file stating why it is there and that it is not used. also consider appending or prepending the file name with ignore/deprecated or an appropriate term